src/index.ts

```diff
@@ -84,6 +84,7 @@ export async function listen(
     port: Number(options_.port),
     verbose: !options_.isTest,
     host: options_.hostname,
+    portRange: [3000, 3100],
```

Comment:
I think we might only assign port range when there is no explicit `options_.port` (same fix of gpp when a nuxt user for example specifies `port` via config or cli flags, they probably are depending on it and don't expect alternative ranges.)

Reply:
Not sure to understand your philosophy here.

Nuxt is already listening to another random port

Reply:
Nuxt is always giving the port option anyway

Reply:
Also we might directly override `alternativePortRange` which is meant for fallback

Reply:
How can we overwite `alternativePortRange` since if we define `port` it's not possible 😅 : https://github.com/unjs/get-port-please/blob/73ced5bdea9fb8e23aef7181ac7338260b1e6f58/src/index.ts#L38

Reply:
[config overrides it](https://github.com/unjs/get-port-please/blob/73ced5bdea9fb8e23aef7181ac7338260b1e6f58/src/index.ts#L41)

Reply:
Pushed `alternativePortRange` instead of `portRange`

Reply:
We just simply need to override gpp to force it keep using alternative port range (because listhen adds it's 3000 default). We could also remove it as an alternative fix.